content/post/80-js-ipfs-0.35.md

```diff
@@ -0,0 +1,240 @@
+---
+date: 2019-04-12
+url: 80-js-ipfs-0-35
+title: js-ipfs 0.35.0 released
+author: Alan Shaw
+---
+
+> Small, acyclic and flexible, just the way you like it
+
+# 🔦 Highlights
+
+## ✂️ Huge bundle size cuts!
+
+One of the biggest goals for JS IPFS is to enable the distributed web in the browser. Part of that is being a good web citizen and minimising the amount of code we ship, to ensure IPFS downloads quicker, `npm install`'s quicker, loads quicker and utilizes fewer resources. It's especially important on resource constrained mobiles and IoT devices.
+
+The good news you want to hear is that we've now chopped around **26%** off the bundle size of JS IPFS since 0.33, that's a reduction of _over_ a quarter of the code we were previously shipping! This is an amazing improvement and a great reason to upgrade.
```

Comment:
Maybe this should include the reduction since 0.34.0 too as the last release?

Reply:
@hugomrdias started work on it when we were at 0.33, so it feels right to compare it against that version. We got some of the benefits in 0.34 but didn't shout about it as it wasn't finished.